datatypes/src/lib.rs

```diff
@@ -24,6 +24,6 @@ pub mod error;
 pub mod operations;
 pub mod plots;
 pub mod primitives;
-pub mod projection;
 pub mod raster;
+pub mod spatial_reference;
```

Comment:
Will there be more functionality in `spatial_reference`? Otherwise we could say
```
mod spatial_reference;

pub use self::spatial_reference::{SpatialReference, SpatialReferenceAuthority, SpatialReferenceOption};
```

Reply:
maybe.... i think we will see when projection operations are added.